goss-testing/scripts/check_bgp_neighbors_established.sh

```diff
@@ -31,6 +31,8 @@ set -o pipefail
 
 TMPFILE=/tmp/check_bgp_neighbors_established.$$.$RANDOM.tmp
 
+SWITCH_PASSWORD={{.Env.SW_ADMIN_PASSWORD}}
```

Comment:
This still won't work.  Bash doesn't know what to do with `{{.Env.SW_ADMIN_PASSWORD}}`

Reply:
ok fixed.  i had thought we had some sort of special setup